src/scss/_functions.scss

```diff
@@ -10,31 +10,27 @@
 	@if (map-has-key($o-colors-palette, $name)) {
 		@return map-get($o-colors-palette, $name);
 	} @else {
-		@warn "Color name '" + $name + "' is not defined in the palette";
-		@return undefined;
+		@warn "Color name '#{inspect($name)}' is not defined in the palette";
+		@return null;
```

Comment:
Because Sass will transform some keywords (colors and other language keywords) if passed as is in an interpolated string.

e.g. if `$name` is `null`, it would output `Color name '' is not defined in the palette`.